atomicapp/providers/openshift.py

```diff
@@ -36,91 +35,205 @@ class OpenShiftProvider(Provider):
     cli = None
     config_file = None
     template_data = None
+    providerapi = "https://127.0.0.1:8443"
```

Comment:
So this is the default? Is that where openshift listens by default if on local machine? - We should probably make this get set in constants.py and used here.